huaweicloud/services/acceptance/ecs/resource_huaweicloud_compute_volume_attach_test.go

```diff
@@ -1,33 +1,65 @@
-package huaweicloud
+package ecs
 
 import (
 	"fmt"
 	"testing"
 
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/ecs"
 
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
```

Comment:
suggest to exchange the locations of the hashicorp packages and the huaweicloud packages.

Reply:
format tools automatically imported, unnecessary to modify.  If you focus on this issue, please use tools to fmt it